bin/add-cmderplugin.ps1

```diff
@@ -0,0 +1,161 @@
+<#
```

Comment:
Why would a user change stuff in bin?  I put it in bin so it would be in 
the path.


On February 20, 2016 2:42:44 PM Jan Schulz <notifications@github.com> wrote:

>> @@ -0,0 +1,161 @@
>> +<#
>
> Again, I don't think this should be installed into `bin`, because bin (the 
> same as config) is under the user's control and therefore changes by the 
> user should not be overwritten, but installing into bin would do this.
>
> On the other hand, I don't see a better dir... The only way would be a 
> `vendor/cmder` dir, which the user either needs to call the script by a 
> absolute name or to add that dir to the path...
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cmderdev/cmder/pull/849/files#r53556940